SalesforceCommerceCloud/pwa-kit
# V3 - Fixes to hybrid login

This PR contains fixes to v3 to make it compatible with Plugin SLAS v7

Changes
* Add 'Bearer ' keyword when calling the OCAPI session bridge
* Call OCAPI session bridge when logging in as registered user
# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

- Generate an app and deploy it to https://test.phased-launch-testing.com (or just go to https://test.phased-launch-testing.com)
- Add an item to your cart (in PWA) and navigate to the cart page (in SFRA)
- Expectation: Item appears in your SFRA cart
- Go back to PWA and login (or create a new account)
- View your cart
- Expectation: Item appears in your SFRA cart. You are logged in SFRA

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
